adhocracy4/comments_async/static/comments_async/comment.jsx

```diff
@@ -201,13 +200,21 @@ export default class Comment extends React.Component {
       }
       comment = (
         <div className={'a4-comments__text' + (this.state.anchored ? ' a4-comments__text--highlighted' : '')}>
```

Comment:
we should change this to a4-comments__text--linked as that what the styling actually shows but we would need to change it everywhere so not sure about doing now?!